src/signalingclient/SignalingClientSubscribe.ts

```diff
@@ -28,6 +29,7 @@ export default class SignalingClientSubscribe {
     public receiveStreamIds: number[],
     public localVideoEnabled: boolean,
     public videoStreamDescriptions: VideoStreamDescription[],
-    public connectionTypeHasVideo: boolean
+    public connectionTypeHasVideo: boolean,
+    public compressedSdpOffer: Uint8Array
```

Comment:
Technically have to make this optional thanks to backwards compat guarantees :D

Reply:
Not sure if it can break compatibility in this case. This is kinda internal and I double anyone extends it.

Reply:
I'm all for that. *technicalllllyyy* its not marked as internal (3.0 idea?) but totally agree no one should be using signaling client raw.